GA/post.py

```
@@ -0,0 +1,1129 @@
+            The optimized blade design data, structured similarly to `reference_design`. 
+            Required if `individual` is an integer. Defaults to None.
+                            
+        Returns
+        -------
+        None
+        """
+        
+        if individual == "opt":
+            # Handle optimum design case
+            if isinstance(res.X, dict):
+                (_, _, optimised_design, _, _) = DesignVectorInterface().DeconstructDesignVector(res.X)
```

Pylint (3.3.7)

[convention] Line too long (103/100)

(C0301)